### PR TITLE
Bump image dependency version

### PR DESCRIPTION
**Pull request by fotiDim**
Solves https://github.com/Baseflow/screenrecorder/issues/39

### CODE REVIEW

**Comment by fotiDim** [comment]
@ueman can you please have a look?

**Comment by ueman** [comment]
Thanks! Please add a changelog entry and then it's good to go

**Comment by fotiDim** [comment]
@ueman done
Can you expect a release soon?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## [0.2.1]
 
 * `Exporter` class exposes now `frames`.
+*  Bump `image` dependency to ^4.0.17
```
Comment (ueman) [inline CHANGELOG.md:4]: This should be in a new release, not in an existing one. You also need to increase the version

**Comment by ueman** [comment]
> @ueman done Can we expect a release soon?

I can't do a release, so @martijn00 or one of his colleagues has to deal with it

File: CHANGELOG.md (hunk 1 above)
Comment (fotiDim) [reply CHANGELOG.md:4]: How so? `0.2.1` is not out yet. On pub.dev I only see `0.2.0`.

Comment (ueman) [reply CHANGELOG.md:4]: Oh interesting, in that case it's okay I assume. Thanks for clarifying

**Comment by martijn00** [comment]
@ueman our team is picking up the pace with our open source plugins. I hope they get to this one soon!

@mvanbeusekom @JeroenWeener @TimHoogstrate

**Comment by fotiDim** [comment]
@martijn00 kind reminder for a release.

**Comment by fotiDim** [comment]
There were some API calls of Image that needed to be adapted. I haven't realized that in this PR so made another one. Note the [new PR](https://github.com/Baseflow/screenrecorder/pull/42) is done against `main` not `develop`.

**Pull request merged**